engine/src/test/java/org/camunda/bpm/engine/test/bpmn/scripttask/ScriptTaskTest.java

```diff
@@ -259,11 +259,11 @@ public void testJavascriptFunctionInvocation() {
     // the variable is defined
     Object variable = runtimeService.getVariable(pi.getId(), "foo");
 
-    if (variable instanceof Double) {
-      // jdk 6/7 - rhino returns Double 3.0 for 1+2
+    if (variable instanceof Integer) {
+      // jdk8+ - nashorn returns Integer 3 for 1+2
       assertEquals(3.0, variable);
-    } else if (variable instanceof Integer) {
-      // jdk8 - nashorn returns Integer 3 for 1+2
+    } else
+      // for previous jdk - nashorn returns Integer 3 for 1+2
       assertEquals(3, variable);
     }
```

Comment:
If JDK 6/7 returned a double and JDK 8+ return Integer, can't we just simplify this whole block to
```java
      assertEquals(3, variable);
```
since we're expecting JDK8+ from now on?